### PR TITLE
[RFR] Admin Musclegroup

### PR DESCRIPTION
**Pull request by BennoDev**
Musclegroups: 
- [x] Create
- [x] Read
- [x] Update
- [x] Delete
- [x] Layout

### CODE REVIEW

**Comment by BennoDev** [comment]
Restarted the crud branches because we needed new DAL changes and rebasing was a mess.

**Comment by BennoDev** [comment]
This PR contains a lot of DAL fixes, decided to do them here because creating a separate PR and rebasing was more likely to cause issues.

**Comment by BennoDev** [comment]
CRUD functionality done, we should use this branch to clean up some layouting so we can use this resource as a starting point for the rest

File: zwekness_api/zwekness_admin/Views/MuscleGroup/List.cshtml [hunk 1]
```diff
@@ -27,7 +23,8 @@
                             Description
                         </th>
                         <th>
-
+                            <a class="btn btn-lg btn-success"
```
Comment (BennoDev) [inline zwekness_api/zwekness_admin/Views/MuscleGroup/List.cshtml]: A button doesn't belong in a table header, as it doesn't describe the tables contents.

Comment (SeppeDev) [inline zwekness_api/zwekness_admin/Views/MuscleGroup/List.cshtml]: Indeed. It's not there anymore now. Added scss files to position it properly.

File: zwekness_api/zwekness_admin/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -9,6 +9,7 @@
     <environment names="Development">
         <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
         <link rel="stylesheet" href="~/css/site.css" />
+        <link rel="stylesheet" href="~/css/styles.css" />
```
Comment (BennoDev) [inline zwekness_api/zwekness_admin/Views/Shared/_Layout.cshtml]: If this file is being minified call it styles.min.css

File: zwekness_api/zwekness_admin/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -69,6 +70,7 @@
                 integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa">
         </script>
         <script src="~/js/site.min.js" asp-append-version="true"></script>
+        <script src="~/js/scripts.js" asp-append-version="true"></script>
```
Comment (BennoDev) [inline zwekness_api/zwekness_admin/Views/Shared/_Layout.cshtml]: If this file is being minified call it scripts.min.js

File: zwekness_api/zwekness_admin/wwwroot/_prototype/scss/edit.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿.c-edit {
+    border: 2px solid $primary;
+    margin-top: 50px;
```
Comment (BennoDev) [inline zwekness_api/zwekness_admin/wwwroot/_prototype/scss/edit.scss]: We should use a map and a function to get values such as this: https://codepen.io/jakealbaugh/post/using-sass-functions-to-access-complex-variable-maps

Comment (BennoDev) [inline zwekness_api/zwekness_admin/wwwroot/_prototype/scss/edit.scss]: This can be used for font sizes, layout values(margins/paddings), colors, etc...

**Pull request merged**